NSubstitute.DbConnection/NSubstitute.DbConnection.Dapper.Tests/DapperQueryTests.cs

```diff
@@ -49,4 +50,18 @@ public void ShouldMockQueryUsingRecordType()
         result[0].Key.Should().Be(1);
         result[0].Value.Should().Be("abc");
     }
+
+    [Test]
+    public void ShouldMockQueryThrow()
+    {
+        var mockConnection = Substitute.For<IDbConnection>().SetupCommands();
+        mockConnection.SetupQuery("delete from table")
+            .Throws(new Exception());
+
+        using var command = mockConnection.CreateCommand();
+        command.CommandText = "delete from table";
+        mockConnection.Open();
+
+        Assert.Throws<Exception>(() => { mockConnection.Query("delete from table"); });
```

Comment:
I'd prefer to use FluentAssertions here and elsewhere to assert that the exception is thrown. Also, the braces around the lambda expression are redundant I think